File: Modul 6/SOAL 4/C/PRAK604.c
#include <stdio.h>
#include <string.h>
#define MAX 100

int main() {
    char input1[MAX], input2[MAX];
    int elemen1, elemen2;
    int asterisk = 0, sharp = 0;

    scanf(" %[^\n]%*c", &input1);
    scanf(" %[^\n]%*c", &input2);
    printf("\n");

    elemen1 = strlen(input1);
    elemen2 = strlen(input2);

    if (elemen1 != elemen2) {
        printf("Panjang kalimat berbeda, pesan palsu");
    } else {
        for (int i = 0; i < elemen1; i++) {
            if (input1[i] == input2[i]) {
                if (input1[i] != ' ') {
                    printf("*");
                    asterisk++;
                } else {
                    printf(" ");
                }
            } else {
                    printf("#");
                    sharp++;
                }
            }
        printf("\n* = %d", asterisk);
        printf("\n# = %d", sharp);

        if (asterisk >= sharp) {
            printf("\nPesan Asli");
        } else {
            printf("\nPesan Palsu");
        }
        }
    return 0;
}

File: Modul 6/SOAL 4/PYTHON/PRAK604.py
input1 = input()
input2 = input()

elemen1 = len(input1)
elemen2 = len(input2)

asterisk = 0
sharp = 0

print()

if (elemen1 != elemen2):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for i in range (elemen1):
        if (input1[i] == input2[i]):
            if (input1[i] != ' '):
                print("*", end='')
                asterisk += 1
            else:
                print(" ", end='')
        else:
            print("#", end='')
            sharp += 1
            
    print("\n* =", asterisk)
    print("# =", sharp)

    if (asterisk >= sharp):
        print("Pesan Asli")
    else:
        print("Pesan Palsu")